multisig/proposals/ethereum/vip-032/index.ts

```diff
@@ -0,0 +1,27 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { makeProposal } from "../../../../src/utils";
+
+export const LIQUID_STAKED_ETH_COMPTROLLER = "0xF522cd0360EF8c2FF48B648d53EA1717Ec0F3Ac3";
+export const vweETH = "0xb4933AF59868986316Ed37fa865C829Eba2df0C7";
+export const vweETH_SUPPLY_CAP = parseUnits("50000", 18);
+export const vweETH_BORROW_CAP = parseUnits("25000", 18);
+export const vstETH = "0x4a240F0ee138697726C8a3E43eFE6Ac3593432CB";
+export const vstETH_BORROW_CAP = parseUnits("10000", 18);
```

Comment:
The market is for the token `wstETH`, so the variable names could be `vwstETH` and `vwstETH_BORROW_CAP`